src/internal_events/prometheus.rs

```diff
@@ -0,0 +1,52 @@
+use super::InternalEvent;
+use crate::sources::prometheus::parser::ParserError;
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct PrometheusRequestCompleted;
```

Comment:
minor nit: but maybe consider making this `PrometheusRequestCompleted {}` to kinda align with our other structs. I personally find plain structs kinda confusing when reading code.